geode-docs/developing/partitioned_regions/colocating_partitioned_region_data.html.md.erb

```diff
@@ -19,27 +19,19 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-By default, <%=vars.product_name%> allocates the data locations for a partitioned region independent of the data locations for any other partitioned region. You can change this policy for any group of partitioned regions, so that cross-region, related data is all hosted by the same member. This colocation speeds queries and other operations that access data from the regions.
+By default, <%=vars.product_name%> allocates the data locations for a partitioned region independent of the data locations for any other partitioned region. You can change this policy for any group of partitioned regions, so that cross-region, related data is all hosted by the same member.
+Colocation is required for some operations,
```

Comment:
This paragraph also appears in overview_custom_partitioning_and_data_colocation.html, so you need to repeat this edit in that file, too.